File: C++/ex.1180.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    int N, menor=0, posicao=0;

    cin >> N;

    int vetor[N];

    for (int i=0; i<N; i++){
        cin >> vetor[i];
    }

    for (int i=0; i<N; i++){
        if (i==0){
            menor=vetor[i];
        }

        if(vetor[i]<menor){
            menor=vetor[i];
            posicao=i;
        }
    }

    cout << "Menor valor: " << menor << endl;
    cout << "Posicao: " << posicao << endl;

    return 0;
}

File: Python/Beecrowd/ex.1180.py
n = int(input())
#inicializando uma lista em python
x = []
num = input().split()

#range() itera sobre um conjunto iniciando do zero e incrementando 1 ate o ultimo elemento (by default)
for i in range(n):
    x.append(int(num[i]))

menor = x[0]

for i in x:
    if i < menor:
        menor = i

print('Menor valor:', menor)
#procurando o indice do menor elemento da lista
print('Posicao:', x.index(menor))